component/src/test/java/org/wso2/extension/siddhi/io/http/source/HttpConflictsTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Wrong license year

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139